hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanActionExecutor.java

```diff
@@ -257,7 +257,8 @@ public HoodieCleanMetadata execute() {
       }
 
       for (HoodieInstant hoodieInstant : pendingCleanInstants) {
-        if (table.getCleanTimeline().isEmpty(hoodieInstant)) {
+        if (table.getCleanTimeline().isEmpty(CleanerUtils.getCleanRequestInstant(table.getMetaClient(), hoodieInstant))) {
+          // remove the empty instant
```

Comment:
we also need to delete the empty inflight file?

Reply:
> we also need to delete the empty inflight file?

The original logic here is to delete the inflight file. I think what you should be saying is whether we also need to delete the request file, right?
I originally attempted to make this change, but I found that it would disrupt the existing ut logic, you can refer to my previous commit: https://github.com/apache/hudi/pull/13192/commits/16230a47b0dfd3433f764d8fe94bbbd959c9db9c. To reduce the unknown risks brought by the change, this change only makes alterations to the way the plan is obtained and does not change any other logic.

Reply:
> we also need to delete the empty inflight file?

Also, we can no longer perform the `isEmpty` judgment on the inflight file now because this file no longer stores the clean plan. Its size is always 0 and the `isEmpty` method is always true.